### PR TITLE
Add a draft conclusion to the ehrQL tutorial

### PR DESCRIPTION
**Pull request by StevenMaude**
This adds a brief summary and next steps for someone completing the tutorial.

### CODE REVIEW

File: docs/ehrql-new-tutorial-conclusion.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# ehrQL tutorial: Conclusion
+
+---8<-- 'includes/data-builder-danger-header.md'
+
+## Summary
+
+Thank you for taking the time to work through this tutorial!
```
Comment (CarolineMorton) [inline docs/ehrql-new-tutorial-conclusion.md:7]:
```suggestion
Thank you for taking the time to work through this tutorial. We hope that you found it useful and have the basic tools to start to work with Data Builder and OpenSAFELY. 
```

File: docs/ehrql-new-tutorial-conclusion.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* How to run filtering and aggregation queries
+
+## Where to look next
+
+Now that you have completed this tutorial,
+you may find the following resources useful
+as reference for writing your own dataset definitions:
+
+* The [Contracts reference](contracts-reference.md)
+* The [ehrQL reference](ehrql-reference.md)
```
Comment (CarolineMorton) [inline docs/ehrql-new-tutorial-conclusion.md:38]: Put this first

File: docs/ehrql-new-tutorial-conclusion.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Existing dataset definitions in the `opensafely` GitHub organisation
```
Comment (CarolineMorton) [inline docs/ehrql-new-tutorial-conclusion.md:39]: Link out to a couple of good examples, when we have them.

Comment (StevenMaude) [reply docs/ehrql-new-tutorial-conclusion.md:39]: Yep, I added a todo to that effect; hopefully we have a working study (or studies) in the main organisation soon.

File: docs/ehrql-new-tutorial-conclusion.md (hunk 2 above)
Comment (StevenMaude) [reply docs/ehrql-new-tutorial-conclusion.md:38]: Done; thanks!

**Pull request merged**